logic/transaction.js

```diff
@@ -783,6 +783,12 @@ class Transaction {
 	 * @todo Add description for the params
 	 */
 	apply(transaction, block, sender, cb, tx) {
+		if (exceptions.inertTransactions.indexOf(transaction.id) > -1) {
```

Comment:
While we are using ES6 around the application, we should use some of its benefits. I suggest to update this to an array builtin method; 

```js
if ( exceptions.inertTransactions.includes(transaction.id) )
```

On other places as well.

Reply:
Thanks!